demo/public/index.html

```diff
@@ -33,7 +33,7 @@
 		<noscript>
 			You need to enable JavaScript to run this app.
 		</noscript>
-		<div id="app" class="wrapper theme-1-active pimary-color-green">
+		<div id="app" class="wrapper theme-1-active primary-color-green">
```

Comment:
@Aringoaway I think you can remove whole `class` from the div, since it should not have any impact (please test it)